src/components/VtToastContainer.vue

```diff
@@ -89,7 +89,12 @@ export default Vue.extend({
       }
     },
     addToast(params: ToastOptionsAndRequiredContent) {
-      const props = Object.assign({}, this.defaults, params);
+      const props = Object.assign(
+        {},
+        this.defaults,
+        (params.type && this.defaults[params.type]) || {},
+        params
+      );
```

Comment:
If you agree with the idea below, this should be enough:
```suggestion
      const props = Object.assign(
        {},
        this.defaults,
        params.type &&
          this.defaults.toastDefaults &&
          this.defaults.toastDefaults[params.type],
        params
      );
```